src/components/Common/Module/FieldPicker.vue

```diff
@@ -153,6 +266,13 @@ export default {
       }
     },
   },
+
+  methods: {
+    filteredFields (fields) {
```

Comment:
Turn this into two computed properties, please. One for available other for selected fields.
You can keep this method, and rename it to filterFields. Then call this method in the computed. 
Reactivity can fail with methods, while computed properties are meant for that